### PR TITLE
refacto upload view

### PR DESCRIPTION
**Pull request by alavieille**

### CODE REVIEW

File: MediaAdminBundle/Resources/public/template/Media/uploadPreview._tpl.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        <div class="title">
+            <span>
+                <input type="text" class="form-control" name="title" value="<%- name %>" />
+            </span>
+            <div class="edit pull-right">
+                <input class="form-control delete-checkbox" type="checkbox" data-file-id="<%- id%>" id="checkbox-<%- id%>">
+                <label for="checkbox-<%- id%>"></label>
+            </div>
+
+            <div class="dossier form-group">
```
Comment (ngilain) [inline MediaAdminBundle/Resources/public/template/Media/uploadPreview._tpl.html]: @alavieille : it's not really an international class name ^^'

File: MediaAdminBundle/Resources/translations/interface.en.yml [hunk 2]
```diff
@@ -56,13 +56,11 @@ open_orchestra_media_admin:
...
-        cancel       : Cancel upload
-        pre_upload   : preparing upload
         failed       : "file could not be uploaded, "
         server_error : "there was a server error: please try again later"
         remaining    : remaining
         processing   : server is processing, please wait
         completed    : completed
-
+        select_media_folder: Select a folder
+        save: Sauvegarder
```
Comment (ngilain) [inline MediaAdminBundle/Resources/translations/interface.en.yml]: @alavielle: I think you meant "Save"

**Comment by alavieille** [comment]
@ngilain pr upadted

**Comment by veolys-nanne** [comment]
:+1:

**Pull request merged**